src/devices/ctm.ts

```diff
@@ -1029,26 +1029,68 @@ export const definitions: DefinitionWithExtend[] = [
         },
         exposes: [e.power(), e.current(), e.voltage(), e.switch(), e.energy()],
     },
-    {
-        zigbeeModel: ["mKomfy Tak"],
-        model: "mKomfy_Tak",
-        vendor: "CTM Lyng",
-        description: "Temperature sensor",
-        extend: [m.battery(), m.temperature()],
-    },
     {
         zigbeeModel: ["mKomfy 2.0"],
         model: "6254380",
         vendor: "CTM Lyng",
         description: "2.0 Stove guard",
         extend: [
             m.deviceTemperature(),
-            m.iasZoneAlarm({zoneType: "alarm", zoneAttributes: ["alarm_1", "alarm_2", "tamper", "battery_low"]}),
+            m.iasZoneAlarm({
+                zoneType: "generic",
```

Comment:
```suggestion
                zoneType: "alarm",
```

Maybe this is better?

Reply:
I used 'generic' because 'alarm' overrides one of the manufacturerZoneAttributes.